exporter/awsxrayexporter/README.md

```diff
@@ -26,12 +26,7 @@ propagated by them using the `X-Amzn-Trace-Id` HTTP header. However, other gener
 supported by replacing fully-random Trace IDs with X-Ray formatted Trace IDs where necessary:
 
 > AWS X-Ray IDs are the same size as W3C Trace Context IDs but differ in that the first 32 bits of a Trace ID
-> is the Unix epoch time when the trace was started. Since X-Ray only allows submission of Trace IDs from the
-> past 30 days, received Trace IDs are checked and spans without a valid timestamp are dropped.
-
-This means in order for spans to appear in X-Ray, the client SDK MUST use an X-Ray ID generator. For more
-information, see
-[configuring the X-Ray exporter](https://aws-otel.github.io/docs/getting-started/x-ray#configuring-the-aws-x-ray-exporter).
+> is the Unix epoch time when the trace was started.
```

Comment:
Do we want to mention although the collector stop validating timestamp in trace id, X-Ray API will continue to validate it until it supports trace id with random number in the timestamp part of id. Meaning customer still need to use X-Ray id generator until service side support launches.

Reply:
Good point, for now I can mention that ID generator is still necessary. In the future, we will need to update this README once we actually don't need it.

Reply:
Change is removed in latest commit. This change will be added once the new `remove timestamp validation` logic is enabled by default